CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 2.5.1
+
+* Adopted source files and demo project for Swift 2.3
```

Comment:
Please end this line with a full stop and two spaces (so that the rendered markdown inserts a newline there while still staying in the same paragraph)